tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -166,6 +169,19 @@ Verify RHODS Release Version Number
     ${version} =  Get RHODS Version
     Should Match Regexp    ${version}    ^[0-9]+\.[0-9]+\.[0-9]+\(-[0-9]+)*$
 
+Verify That Alert Notification Emails Can Be Updated And Changed In Alertmanager Configuration
+    [Documentation]    Vrifies the Alert Notification email is updated in Addon-Managed-Odh-Parameters Secret and Alertmanager ConfigMap
+    [Tags]    Sanity
```

Comment:
I think Sanity tag should be removed, as it's already in Tier2

If you add tag  Deployment-AddOnFlow this test can be ignored in PSI